jigasi/Dockerfile

```diff
@@ -1,11 +1,13 @@
 ARG JITSI_REPO=jitsi
 FROM ${JITSI_REPO}/base-java
 
+ENV GOOGLE_APPLICATION_CREDENTIALS /config/key.json
+
 RUN \
 	apt-dpkg-wrap apt-get update && \
-	apt-dpkg-wrap apt-get install -y jigasi && \
+	apt-dpkg-wrap apt-get install -y jigasi jq && \
 	apt-cleanup
 
 COPY rootfs/ /
 
-VOLUME /config
+VOLUME ["/config", "/tmp/transcripts"]
```

Comment:
side question, shouldn't we have this in / or /config? /tmp doesn't seem like the right place for it...

Reply:
As I applied the changes for this PR I just noticed that he transcripts are saved inside the container without having documented the directories as volumes. So I just added them but yeah they are not at the right place in this PR. I will remove them.